docs/api/README.md

```diff
@@ -356,7 +356,7 @@ Since it's just a component, it works with `<transition>` and `<keep-alive>`. Wh
   })
 
   router.beforeEach((to, from) => {
-    if (from === START_LOCATION) {
+    if (from === router.START_LOCATION) {
```

Comment:
It should be static property, so
```suggestion
    if (from === Router.START_LOCATION) {
```

Can you update zh and ru files as well in the docs folder?

Reply:
Maybe also change `Router` to `VueRouter`? Because now we have `router` and `Router` which are confusing